### PR TITLE
Pushing Week 8 + 9 blog

### PR DESCRIPTION
**Pull request by shilpiprd**

### CODE REVIEW

File: doc/posts/2023/2023_07_27_Shilpi_week8.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Generating Fit Functions : Week 9
```
Comment (skoudoro) [inline doc/posts/2023/2023_07_27_Shilpi_week8.rst:1]: PR say week 8+9
filename say week8
title say week 9

Can you update that and make it more consistent.....

**Review by skoudoro — CHANGES_REQUESTED** [review]
Before a deeper review, can you 

- check again all the markup:
  * filename, code, function name, etc with backtick
  * remove all the underscore of design matrix
- make shorter sentences to improve readability

Thanks

**Comment by shilpiprd** [comment]
So, I improved whatever I thought could be improved. But please let me know if there's anything else I could improve.

**Review by skoudoro — APPROVED** [review]
Thanks for the reshaping. Looks good, merging

**Pull request merged**